bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/PilightBindingProvider.java

```diff
@@ -35,9 +35,9 @@
 	 * can be bound to multiple items in openHAB. 
 	 * 
 	 * @param instance Name of the pilight instance 
-	 * @param location Location of the device in pilight
 	 * @param device Name of the device in pilight
 	 * @return The binding configs for the pilight references or empty list if not found
 	 */
-	public List<PilightBindingConfig> getBindingConfigs(String instance, String location, String device);
+	public List<PilightBindingConfig> getBindingConfig(String instance, String device);
```

Comment:
why did you chance the name to singular? A list is returned here so the method name should reflect that properly